airflow/www/views.py

```diff
@@ -2288,6 +2292,53 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except :
+            pass
+
+        # Get the Git repo and git hash : first try pip freeze, then look at git
+        git_repo = git_hash = None
+        try:
+            p = subprocess.Popen('pip freeze | grep airflow',
```

Comment:
I understand I'm just not sure it's a good idea to create a dependency on PIP/git in the code as it violates separation of concerns (the code shouldn't know anything about what service it is deployed with), might be worth getting some more opinions on this. Also there is the issue with pip returning the wrong version (e.g. due to PYTHONPATH being off or airflow being run in a virtualenv). Same goes for the git dependency (unless we can move that to the deploy process).

Would be useful to get some more opinions on this:
cc @criccomini @plypaul @jlowin